libs/pages/application/src/lib/page-application.tsx

```diff
@@ -23,6 +31,36 @@ function PageApplicationWrapped() {
 
   useDocumentTitle(`${service?.name || 'Application'} - Qovery`)
 
+  const { setQuickActions } = useContext(SpotlightContext)
+  useEffect(() => {
```

Comment:
I understand the point to have the possibility to set quick actions outside the Spotlight but I am afraid to maintain it. For instance, here we need to duplicate this behavior 3 times (page-database, page-application, and logs) to have the same thing.

Wouldn't it be simpler to be able to maintain this from the Spotlight domain? Today we don't have a lot of actions but tomorrow with complex platform, should be more simpler to have this quick actions on the same spot.

Reply:
Yes I agree with you, I was struggling with react-router available APIs that's why I didn't do it .
I've managed to simplify it this way:
https://github.com/Qovery/console/pull/1265/commits/5fc71a9b29304a6c20eef7d636e1b19df32b3418